module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -658,39 +673,22 @@ public function getHolding($bibId, array $patron = null, array $options = [])
      */
     protected function getDueDate($itemId, $showTime)
     {
-        $dueDateValue = '';
-        $query = [
-            'query' => 'itemId==' . $itemId
-        ];
+        $query = 'itemId==' . $itemId;
         foreach ($this->getPagedResults(
             'loans',
             '/circulation/loans',
-            $query
+            compact('query')
```

Comment:
Used compact to build an array here so we don't have to build it "by hand" above.